specs/_features/eip7732/p2p-interface.md

```diff
@@ -215,9 +216,44 @@ _ _[IGNORE]_ `header.parent_block_root` is the hash tree root of a known beacon
 | `DENEB_FORK_VERSION`     | `deneb.SignedBeaconBlock`     |
 | `EIP7732_FORK_VERSION`   | `eip7732.SignedBeaconBlock`   |
 
-##### BeaconBlocksByRoot v3
+##### ExecutionPayloadEnvelopesByRange v1
 
-**Protocol ID:** `/eth2/beacon_chain/req/beacon_blocks_by_root/3/`
+**Protocol ID:** `/eth2/beacon_chain/req/execution_payload_envelopes_by_range/1/`
+
+*[New in EIP-7732]*
+
+Request Content:
+
+```
+(
+  start_slot: Slot
+  count: uint64
+)
+```
+
+Response Content:
+
+```
+(
+  List[SignedExecutionPayloadEnvelope, MAX_REQUEST_BLOCKS_DENEB]
+)
+```
+
+Specifications of req\response methods are equivalent to [BeaconBlocksByRoot v2](#beaconblocksbyroot-v2), with the only difference being the response content.
```

Comment:
Wouldn't equivalent to BeaconBlocksByRange makes more sense here?

Reply:
Yes, good catch. Also, we should move this section down, immediately before ExecutionPayloadEnvelopesByRoot. And delete the `blobsidecarsbyroot-v1` section which doesn't add anything new.

```diff
  [BeaconBlocksByRange v2](#beaconblocksbyrange-v2)
- [ExecutionPayloadEnvelopesByRange v1](#executionpayloadenvelopesbyrange-v1)
  [BeaconBlocksByRoot v2](#beaconblocksbyroot-v2)
- [BlobSidecarsByRoot v1](#blobsidecarsbyroot-v1)
+ [ExecutionPayloadEnvelopesByRange v1](#executionpayloadenvelopesbyrange-v1)
  [ExecutionPayloadEnvelopesByRoot v1](#executionpayloadenvelopesbyroot-v1)
```

Reply:
ahh 🤦